builtin/mainmenu/content/contentdb.lua

```diff
@@ -93,11 +93,27 @@ local function start_install(package, reason)
 		if result.msg then
 			gamedata.errormessage = result.msg
 		else
+			local delete_old_dir
+			if package.path then
+				local path_array = package.path:split(DIR_DELIM)
+				local name = path_array[#path_array]
+				local name_len = #name
```

Comment:
This substringing is a bit awkward. Why not write this simply as `local name = basename:match("(.*)_game$") or basename`?